lib/lndclient/LndClient.ts

```diff
@@ -66,7 +66,6 @@ class LndClient extends BaseClient {
    */
   constructor(private config: LndClientConfig, logger: Logger) {
     super(logger);
-
```

Comment:
We have blank lines after all calls to `super` in the codebase so we should keep this consistent.